source/languages/en/riak/dev/taste-of-riak/nodejs.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Taste of Riak: NodeJS"
+project: riak
+version: 1.4.0+
+document: guide
+toc: true
+audience: beginner
+keywords: [developers, client, javascript, nodejs]
+---
+
+If you haven't set up a Riak Node and started it, please visit the [[Prerequisites|Taste of Riak: Prerequisites]] first.
+
+To try this flavor of Riak, a working installation of NodeJS 0.12 or later is required.
+
+Code for these examples is available [here](https://github.com/basho/riak-nodejs-client-examples/blob/master/dev/taste-of-riak/introduction.js). To run, follow these directions:
```

Comment:
I guess it's a six of one, half-dozen of the other situation, then. No reason to change what's written to replace it with an equally fragile solution.